docs/root/start/quick-start/configuration-dynamic-filesystem.rst

```diff
@@ -90,3 +95,8 @@ proxies over ``TLS`` to https://www.envoyproxy.io.
     :language: yaml
     :linenos:
     :emphasize-lines: 11, 17-18, 22-23
+
+.. seealso::
+
+   :ref:`Atomic swap <_config_runtime_symbolic_link_swap>`
```

Comment:
im not sure if this is immediately clear as to the relevance

perhaps we can change the link title to something like

> Atomic swap for configuration file updates

Reply:
I still think this should just be ref linked above. It seems fine to do inline.

/wait

Reply:
@mattklein123 im also a bit concerned that this link is confusing (esp to a beginner) as when i click through its related to the runtime fs and there isnt any information there about updating configuration files

Reply:
OK whatever you think this is best is fine with me. I was thinking something along the lines of: "Envoy will only watch configuration files for atomic moves. An example of how to set this up is described [here]."

Reply:
my suggestion would be:

- add a bit more info wrt cp/mv and why the sed example works (re https://github.com/envoyproxy/envoy/pull/20475#issuecomment-1079019891 and following comment)
- move the link to see also at the bottom